testing/todocheck_test.go

```diff
@@ -802,3 +802,644 @@ func TestPrintingVersionFlagStopsProgram(t *testing.T) {
 		t.Errorf("%s", err)
 	}
 }
+
+func TestScriptsCustomTodos(t *testing.T) {
```

Comment:
Could you extract all new tests from this file into a `customtodos_test.go`?

Just to keep files small